libsatadrivers/LICENSE_BSD2.txt

```diff
@@ -0,0 +1,30 @@
+Files described as being under the "BSD 2-Clause" license fall under the
```

Comment:
This file should not be included, instead there should be SPDX tags in the sources, and the corresponding license files under `LICENSES/`

Reply:
Right, will do. Also have to fix up all files' licenses as well.